metadata-models/src/main/pegasus/com/linkedin/externalrolemetadata/ExternalRoleProvisionedUser.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.externalrolemetadata
+
+import com.linkedin.common.Urn
+/**
+ * Provisioned users of a role
+ */
+@Aspect = {
+  "name": "externalRoleProvisionedUser"
```

Comment:
What will happen when the requirement to add groups comes along? How about other nested roles? 

I'd rename this aspect to "actors", so that we can support users, groups, and nested roles

Reply:
Have renamed to actors and now we have the ability to add groups under actors when we require